moveit_kinematics/cached_ik_kinematics_plugin/config/kdl_cached.yaml

```diff
@@ -0,0 +1,75 @@
+arm:
```

Comment:
This belongs in the https://github.com/ros-planning/moveit_kinematics_tests/ repo